specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/examples/ManagedEnvironmentMaintenanceConfigurationsGet.json

```diff
@@ -0,0 +1,32 @@
+{
+    "parameters": {
+      "environmentName": "managedEnv",
+      "resourceGroupName": "rg1",
+      "configName": "default",
+      "api-version": "2024-10-02-preview",
+      "subscriptionId": "subid"
+    },
+    "responses": {
+      "200": {
+        "body": {
+          "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.App/managedEnvironments/managedEnv/maintenanceConfigurations/default",
+          "location": "East US",
+          "name": "managedEnv/default",
+          "type": "Microsoft.App/ManagedEnvironments/MaintenanceConfigurations",
+          "properties": {
+            "scheduleEntries": [
+              {
+                "dayOfWeek": "Monday",
+                "startHourUtc": 12,

```

Comment:
Should it is be `startHour` to have extensibility in future to support utcOffset